crates/rpc-types/src/eth/transaction/common.rs

```diff
@@ -17,3 +18,17 @@ pub struct TransactionInfo {
     /// Base fee of the block.
     pub base_fee: Option<u128>,
 }
+
+impl From<Transaction> for TransactionInfo {
+    fn from(tx: Transaction) -> Self {
```

Comment:
since this is small, could we also do `From<&Transaction>` or do we always want to clone?

Reply:
Yes you are right, in this case I think `From<&Transaction>` is better and `From<Transaction>` would be useless, just fixed